kf/bpf.go

```diff
@@ -314,11 +314,21 @@ func (b *BPF) Stop(ifaceName, direction string, chain bool) error {
 	}
 
 	// unload the BPF programs
-	if b.ProgMapCollection != nil {
-		if err := b.UnloadProgram(ifaceName, direction); err != nil {
-			return fmt.Errorf("BPFProgram %s unload failed on interface %s with error: %w", b.Program.Name, ifaceName, err)
+	allInterfaces, _ := getHostInterfaces()
+	for iface := range allInterfaces {
```

Comment:
We should fetch interface from map and avoid for loop.

Reply:
fixed